templates/ilvo/ILVO data submission.json

```diff
@@ -0,0 +1,73 @@
+{
+    "metadata": {
+      "name": "nucleic acid sequencing",
```

Comment:
Should be best to change both `group` and `name` to indicate ILVO and `run data submission`, since it is only about submission of a run.

Reply:
ok